mega-zep-frontend/src/main/angular/frontend/src/app/modules/project-management/services/project-management.service.ts

```diff
@@ -12,9 +12,9 @@ export class ProjectManagementService {
   constructor(private configService: ConfigService, private httpClient: HttpClient) {
   }
 
-  getEntries(year: number, month: number): Observable<Array<ProjectManagementEntry>> {
```

Comment:
create(comment: string, yearMonth: string, projectName: string) {
    let params = new HttpParams()
      .append("all", "true")


    return this.httpClient.post<ProjectComment>(
      this.configService.getBackendUrlWithContext('/projectcomments'),
      new ProjectComment(comment, yearMonth, projectName), {params: params});
  }